src/components/InpageNav/InpageNav.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+type InpageNavItemLinksProps = {
+  links?: InpageNavItemProps[];
+};
+
+type InpageNavItemProps = {
+  title?: string;
+  id?: string;
+};
+
+export const InpageNav = ({ links }: InpageNavItemLinksProps) => {
+  return (
```

Comment:
Minor: this works but from a tidiness PoV, as there are no `const` declarations or other javascript going on in this function you could simplify with implicit return i.e.
```
export const InpageNav = ({ links }: InpageNavItemLinksProps) => (
  {/* JSX goes here */}
)
```